dietpi/dietpi-software

```diff
@@ -12689,6 +12723,15 @@ If no WireGuard (auto)start is included, but you require it, please do the follo
 			[[ -d '/usr/lib/x86_64-linux-gnu' ]] && G_EXEC rmdir --ignore-fail-on-non-empty /usr/lib/x86_64-linux-gnu
 		fi
 
+		if To_Uninstall 206 # Moonlight
+		then
+			G_AGP moonlight-embedded
+			local gpg_keyring_path="/usr/share/keyrings/moonlight-game-streaming-moonlight-embedded-archive-keyring.gpg"
+			local repo_path="/etc/apt/sources.list.d/moonlight-game-streaming-moonlight-embedded.list"
+			[[ -f "$gpg_keyring_path" ]] && G_EXEC rm "$gpg_keyring_path"
+			[[ -f "$repo_path" ]] && G_EXEC rm "$repo_path"
```

Comment:
I'm no fans of defining variables which are used only 1-2 time 😉:
```suggestion
			[[ -f '/etc/apt/sources.list.d/dietpi-moonligh.list' ]] && G_EXEC rm /etc/apt/sources.list.d/dietpi-moonligh.list
			[[ -f '/etc/apt/trusted.gpg.d/dietpi-moonlight.gpg' ]] && G_EXEC rm /etc/apt/trusted.gpg.d/dietpi-moonlight.gpg
```

Reply:
agreed!